MAD.Procore.StudioHoursUpload/Queries/NamelyStudioHourQuery.sql

```diff
@@ -13,7 +13,7 @@ AS (SELECT [Company email],
         FROM NamelyReportKeyValue
         WHERE LogDateTime =
         (
-            SELECT MAX(LogDateTime)FROM NamelyReportKeyValue
+            SELECT MIN(LogDateTime) FROM NamelyReportKeyValue WHERE LogDateTime >= ISNULL(@LastProcessedDate, '')
```

Comment:
If no parameter is passed through, it should be MAX
No parameter = most up to date log value